bin/run-couchdb.sh

```diff
@@ -15,5 +15,6 @@ echo -n "starting: $NAME "
 docker run $FLAGS \
 	--net $NETWORK \
 	-h $NAME \
+	-v "$(pwd)/couchdb_data:/usr/local/var/lib/couchdb:rw" \
```

Comment:
It seems it's not all that hard, I've commited updates. Probably you want to reflect it in your compose file though.